components/ui/colors/build.gradle

```diff
@@ -18,6 +18,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    buildFeatures {
+        viewBinding true
+    }
```

Comment:
Like this component, it doesn't really have any UI. Why are we adding it to all? There's also an `android {}` block in the root build.gradle file if we want something applied to all modules, but I feel like we only want to do this to selected modules - and over time they probably become less (since we will be using less and less `View`s).